libr/anal/class.c

```diff
@@ -1218,3 +1219,99 @@ R_API void r_anal_class_list_vtable_offset_functions(RAnal *anal, const char *cl
 		ls_free (classes);
 	}
 }
+
+static void free_node_info(void *ptr) {
```

Comment:
Since RGraphNodeInfo is declared in r_graph.h shouldn't this and the constructor be there as well?

Reply:
It could be, but I am not sure. Because in this specific case the free function abides the RListFree signature, so it's untyped with the void *.